asf_core_data/pipeline/mcs/process/process_mcs_installations.py

```diff
@@ -133,11 +151,16 @@ def get_processed_installations_data(refresh=True):
 
     installations_data = get_raw_installations_data(refresh=refresh)
 
+    # we need to change this later to account for other versions
+    historical_installers_processed_data = get_processed_historical_installers_data(
+        "20230207"
+    )
+
     installations_data = add_hp_features(installations_data)
     installations_data = mask_outliers(installations_data)
     installations_data = identify_clusters(installations_data)
-    # installations_data["installer_name"] = installations_data["installer_name"].apply(
-    #     clean_company_name
-    # )
+
+    # Adding variable with unique installer ID
+    get_installer_unique_id(installations_data, historical_installers_processed_data)
```

Comment:
`get_installer_unique_id()` doesn't return anything...? 
Where does the data with added unique ID go? Sorry if I missed something.

Reply:
changed :)